Source/CommandManager.cs

```diff
@@ -218,95 +219,18 @@ public static void Execute(string commandString)
 
             if (!found)
             {
-                Debug.LogError("'" + commandName + "' command exists, but invalid arguments were given." +
-                               arguments.Count);
-                return;
+                Debug.LogError("'" + commandName + "' command exists, but invalid arguments were given." + arguments.Count);
```

Comment:
For better readibility we can use string interpolation. `$"'{commandName}' command exists, but invalid arguments ({arguments.Count}) were given."`. (Apply to all formated strings)